R/check_dhis2_attributes-helpers.R

```diff
@@ -0,0 +1,248 @@
+#' Get the dataset identifiers and names
+#'
+#' @param username the user name
+#' @param password the user's password
+#' @param base_url the base URL of the DHIS2 server
+#'
+#' @export
+#'
+get_data_sets <- function(base_url, username, password) {
+  checkmate::assertCharacter(base_url, len = 1L, null.ok = FALSE,
```

Comment:
According to the docs `len` is the exact expected length. Is `min.chars = 13L` a better check. 13 characters because at the minimum, it will require `http://lll.dd`